src/main/java/rx/internal/util/unsafe/MessagePassingQueue.java

```diff
@@ -29,7 +29,7 @@
  * 
  * @param <M> the event/message type
  */
-interface MessagePassingQueue<M> {
+public interface MessagePassingQueue<M> {
```

Comment:
`javadoc` complained that the spsc queues reference it but it is not visible enough.